src/pre_process/m_global_parameters.fpp

```diff
@@ -445,6 +445,14 @@ contains
             patch_ib(i)%m = dflt_real
             patch_ib(i)%p = dflt_real
             patch_ib(i)%slip = .false.
+
+            ! Proper default values for translating STL models
+            patch_ib(i)%model%scale(:) = 1d0
```

Comment:
i don't think these defaults make sense. what happened to `dflt_real` and `dflt_int`? i guess we also now need `dflt_str`